lib/browser_result.js

```diff
@@ -1,10 +1,11 @@
 'use strict'
 
 class BrowserResult {
-  constructor () {
+  constructor (total = 0) {
```

Comment:
Why this new API?  Please add to description.

Reply:
I introduced it to use it in that situation:
https://github.com/karma-runner/karma/blob/8d302e5f106d3d07efa04f86bc1135fa588fd3ab/lib/browser.js#L73

Reply:
ok good thx